File: hw/pastor/u2/hw19 Inheritance/Javascript/ec.edu.espe.farmsystem.model/cage.js
class Cage {
    constructor(id, description, type, location) {
        this.id = id;
        this.description = description;
        this.type = type; // 1 coop, 2 stable, 3 pens
        this.location = location;
    }

    toString() {
        return `[${this.id}] ${this.description} (${this.location})`;
    }
}

module.exports = Cage;

File: hw/ramos/u2/hw19Inheritance/Python/model/cage.py
class Cage:
    def __init__(self, id, description, type):
        self.id = id
        self.description = description
        self.type = type  # 1 coop, 2 stable, 3 pens

    def __str__(self):
        return (
            f"Cage:\n"
            f"  ID: {self.id}\n"
            f"  Description: {self.description}\n"
            f"  Type: {self.type}"
        )
